packages/plugin-core/src/views/GraphPanel.ts

```diff
@@ -42,6 +42,8 @@ export class GraphPanel implements vscode.WebviewViewProvider {
 
     this.showOutwardLinks =
       MetadataService.instance().graphPanelShowOutwardLinks ?? true;
+
+    this._graphDepth = 1;
```

Comment:
Can we update this to `this.graphDepth = MetadataService.instance().graphDepth || 1;`

Reply:
done!